api.md

```diff
@@ -1007,7 +1007,20 @@ AS
         FROM conditions
         GROUP BY time_bucket('1day', timec), location, humidity, temperature;
 ```
+
+Create an up-to-date continuous aggregate view that runs whenever new data is received.
+```sql
+CREATE VIEW continuous_aggregate_view( timec, minl, sumt, sumh )
+WITH ( timescaledb.continuous,
+    timescaledb.refresh_lag = '5 hours',
+    timescaledb.refresh_interval = '-1h' )
```

Comment:
Which gets set to a negative value, `refresh_lag` or `refresh_interval`?

Reply:
Thanks for catching that - I messed up.